org.qgis.qgis.json

```diff
@@ -0,0 +1,459 @@
+{
+    "id" : "org.qgis.qgis",
+    "branch" : "stable",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "5.11",
+    "sdk" : "org.kde.Sdk",
+    "command" : "qgis",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "separate-locales" : false,
+    "build-options" : {
+        "env" : {
+            "PYTHONPATH" : "/app/lib/python3.5/site-packages:/usr/lib/python3.5/site-packages"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "qt5-qtwebkit",
```

Comment:
You could use https://github.com/flathub/io.qt.qtwebkit.BaseApp like e.g. https://github.com/flathub/org.musescore.MuseScore does, but it is based on 5.9 at the moment.

Reply:
I was explicitly asked by qgis developers to use Qt 5.11.